### PR TITLE
Fix WPCS: Generic.(...)

### PR DESCRIPTION
**Pull request by pedro-mendonca**
- [x] `Generic.Arrays.DisallowShortArraySyntax.Found`
- [x] `Generic.CodeAnalysis.ForLoopWithTestFunctionCall.NotAllowed`
- [x] `Generic.Commenting.DocComment.SpacingAfter`
- [x] `Generic.Commenting.DocComment.SpacingBeforeTags`
- [x] `Generic.ControlStructures.InlineControlStructure.NotAllowed`
- [x] `Generic.Formatting.MultipleStatementAlignment.IncorrectWarning`
- [x] `Generic.Formatting.MultipleStatementAlignment.NotSameWarning`
- [x] `Generic.Functions.FunctionCallArgumentSpacing.NoSpaceAfterComa`
- [x] `Generic.Functions.FunctionCallArgumentSpacing.SpaceBeforeComma`
- [x] `Generic.Functions.FunctionCallArgumentSpacing.TooMuchSpaceAfterComa`
- [x] `Generic.PHP.ForbiddenFunctions.FoundWithAlternative`
- [x] `Generic.PHP.LowerCaseConstant.Found`
- [x] `Generic.WhiteSpace.DisallowSpaceIndent.SpacesUsed`

See #585

### CODE REVIEW

File: gp-includes/routes/project.php [hunk 1]
```diff
@@ -386,8 +390,9 @@ public function permissions_post( $project_path ) {
 				'locale_slug' => gp_post( 'locale' ),
 				'set_slug'    => gp_post( 'set-slug' ),
 			) );
-			if ( $this->invalid_and_redirect( $new_permission, gp_url_current() ) ) return;
-			$permission = GP::$validator_permission->create( $new_permission );
+			if ( $this->invalid_and_redirect( $new_permission, gp_url_current() ) ) { return;
+			}
```
Comment (ocean90) [inline gp-includes/routes/project.php]:
```suggestion
			if ( $this->invalid_and_redirect( $new_permission, gp_url_current() ) ) {
				return;
			}
```

File: gp-includes/routes/translation-set.php [hunk 2]
```diff
@@ -14,10 +14,11 @@
...
 class GP_Route_Translation_Set extends GP_Route_Main {
 	public function new_get() {
-		$set = new GP_Translation_Set;
+		$set             = new GP_Translation_Set;
 		$set->project_id = gp_get( 'project_id' );
-		$project = $set->project_id ? GP::$project->get( $set->project_id ) : null;
-		if ( $this->cannot_edit_set_and_redirect( $set ) ) return;
+		$project         = $set->project_id ? GP::$project->get( $set->project_id ) : null;
+		if ( $this->cannot_edit_set_and_redirect( $set ) ) { return;
+		}
```
Comment (ocean90) [inline gp-includes/routes/translation-set.php]:
```suggestion
		if ( $this->cannot_edit_set_and_redirect( $set ) ) {
			return;
		}
```

File: gp-includes/things/project.php [hunk 3]
```diff
@@ -232,20 +232,22 @@ public function normalize_fields( $args ) {
...
-		if ( $parent_project )
-			$path = gp_url_join( $parent_project->path, $this->slug );
-		elseif ( ! $wpdb->last_error )
-			$path = $this->slug;
-		else
+		if ( $parent_project ) {
+			$path          = gp_url_join( $parent_project->path, $this->slug );
+		} elseif ( ! $wpdb->last_error ) {
+			$path          = $this->slug;
+		} else {
```
Comment (ocean90) [inline gp-includes/things/project.php]: The alignment is incorrect as the lines are not after each other.

**Review by ocean90 — CHANGES_REQUESTED** [review]
Please fix all the lines that are now 
```
{ return;
}
```

or 

```
{ continue;
}
```

There needs to be a line break after the opening bracket.

File: gp-includes/routes/project.php (hunk 1 above)
Comment (pedro-mendonca) [reply gp-includes/routes/project.php]: I have this fix in a commit for `Squiz.ControlStructures.ControlSignature.NewlineAfterOpenBrace`
Should I push to this same PR all the wpcs Squiz.(...) and WordPress.(...) sniff fixes?

Comment (ocean90) [reply gp-includes/routes/project.php]: No, but changes like this should not introduce new code style issues. Please only adjust the lines changed in this PR.

File: gp-includes/things/project.php (hunk 3 above)
Comment (pedro-mendonca) [reply gp-includes/things/project.php]: Indeed, I guess it was formated before the line break.
Fixed.

File: gp-includes/routes/project.php (hunk 1 above)
Comment (pedro-mendonca) [reply gp-includes/routes/project.php]: Ok, fixing it.

File: gp-includes/schema.php [hunk 4]
```diff
@@ -68,15 +70,15 @@ function gp_schema_get() {
 	 */
 
 	/*
- 	 * The maximum length for the slug component of the project_id_slug_locale key is limited by the index size limit
+	  * The maximum length for the slug component of the project_id_slug_locale key is limited by the index size limit
```
Comment (ocean90) [inline gp-includes/schema.php]: Only one whitespace after the tab.

File: gp-includes/schema.php [hunk 5]
```diff
@@ -68,15 +70,15 @@ function gp_schema_get() {
...
 	 * minus the size of the project_id (4 bytes = 1 character) and locale (10 characters).
 	 *
 	 * Also make sure to never go over the length of the column.
 	 */
 	$max_pid_slug_locale_key_length = min( $max_index_length - 1 - 10, 255 );
 
 	/*
- 	 * The maximum length for the slug component of the locale_slug key is limited by the index size limit
+	  * The maximum length for the slug component of the locale_slug key is limited by the index size limit
```
Comment (ocean90) [inline gp-includes/schema.php]: Only one whitespace after the tab.

File: gp-includes/schema.php [hunk 6]
```diff
@@ -108,7 +110,7 @@ function gp_schema_get() {
 	 */
 
 	/*
- 	 * The maximum length for the components of the singular_plural_context key is limited by the index size limit
+	  * The maximum length for the components of the singular_plural_context key is limited by the index size limit
```
Comment (ocean90) [inline gp-includes/schema.php]: Only one whitespace after the tab.

File: gp-includes/schema.php [hunk 7]
```diff
@@ -181,15 +183,15 @@ function gp_schema_get() {
 	 */
 
 	/*
- 	 * The maximum length for the meta_key component of the object_type__meta_key key is limited by the index size limit
+	  * The maximum length for the meta_key component of the object_type__meta_key key is limited by the index size limit
```
Comment (ocean90) [inline gp-includes/schema.php]: Only one whitespace after the tab.

File: gp-includes/schema.php [hunk 8]
```diff
@@ -181,15 +183,15 @@ function gp_schema_get() {
...
 	 * minus the size of the object_type (32 characters).
 	 *
 	 * Also make sure to never go over the length of the column.
 	 */
 	$max_objtype_metakey_key_length = min( $max_index_length - 32, 255 );
 
 	/*
- 	 * The maximum length for the meta_key component of the object_type__object_id__meta_key key is limited by the index size limit
+	  * The maximum length for the meta_key component of the object_type__object_id__meta_key key is limited by the index size limit
```
Comment (ocean90) [inline gp-includes/schema.php]: Only one whitespace after the tab.

**Review by ocean90 — CHANGES_REQUESTED** [review]

File: gp-includes/schema.php (hunk 4 above)
Comment (pedro-mendonca) [reply gp-includes/schema.php]: Fixed in https://github.com/GlotPress/GlotPress-WP/pull/1076/commits/aaf1f070efcc5cb7f040b2d812355ad72d5cc83d

File: gp-includes/schema.php (hunk 5 above)
Comment (pedro-mendonca) [reply gp-includes/schema.php]: Fixed in https://github.com/GlotPress/GlotPress-WP/pull/1076/commits/aaf1f070efcc5cb7f040b2d812355ad72d5cc83d

File: gp-includes/schema.php (hunk 6 above)
Comment (pedro-mendonca) [reply gp-includes/schema.php]: Fixed in https://github.com/GlotPress/GlotPress-WP/pull/1076/commits/aaf1f070efcc5cb7f040b2d812355ad72d5cc83d

File: gp-includes/schema.php (hunk 7 above)
Comment (pedro-mendonca) [reply gp-includes/schema.php]: Fixed in https://github.com/GlotPress/GlotPress-WP/pull/1076/commits/aaf1f070efcc5cb7f040b2d812355ad72d5cc83d

File: gp-includes/schema.php (hunk 8 above)
Comment (pedro-mendonca) [reply gp-includes/schema.php]: Fixed in https://github.com/GlotPress/GlotPress-WP/pull/1076/commits/aaf1f070efcc5cb7f040b2d812355ad72d5cc83d

**Comment by pedro-mendonca** [comment]
@ocean90 why is the last commit failing?
The report on https://travis-ci.org/github/GlotPress/GlotPress-WP/builds/676634828 seems broken somehow

File: gp-includes/routes/project.php [hunk 9]
```diff
@@ -386,8 +390,11 @@ public function permissions_post( $project_path ) {
 				'locale_slug' => gp_post( 'locale' ),
 				'set_slug'    => gp_post( 'set-slug' ),
 			) );
-			if ( $this->invalid_and_redirect( $new_permission, gp_url_current() ) ) return;
-			$permission = GP::$validator_permission->create( $new_permission );
+			if ( $this->invalid_and_redirect(
+				$new_permission, gp_url_current() ) ) {
+					return;
+			}
```
Comment (ocean90) [inline gp-includes/routes/project.php]:
```suggestion
			if ( $this->invalid_and_redirect( $new_permission, gp_url_current() ) ) {
				return;
			}
```

No line break before `$new_permission`.

File: gp-includes/routes/project.php [hunk 10]
```diff
@@ -386,8 +390,11 @@ public function permissions_post( $project_path ) {
...
+			$permission          = GP::$validator_permission->create( $new_permission );
```
Comment (ocean90) [inline gp-includes/routes/project.php]:
```suggestion
			$permission = GP::$validator_permission->create( $new_permission );
```

**Review by ocean90 — CHANGES_REQUESTED** [review]
Do you use [`phpcbf`](https://github.com/squizlabs/PHP_CodeSniffer/wiki/Fixing-Errors-Automatically) to fix the errors?

**Comment by pedro-mendonca** [comment]
Yes I do, but earlier I was doing it by sniff, wich gave me a lot of errors.
Now I've applied phpcfb (`composer format`) and commited changes by sniff.

**Pull request merged**